src/modules/spaces/controllers/spaces.controller.ts

```diff
@@ -45,14 +45,23 @@ export class SpacesController {
 		return spaces;
 	}
 
+	@Get(':slug')
```

Comment:
How does nestjs you know the difference between these 2 requests?
```
http://localhost:3100/spaces/my-slug
http://localhost:3100/spaces/8a8af735-a3f0-4b7f-9b9d-429e0d8fc561
```

I don't think this will be possible. But we no longer need the /spaces/:id endpoint, since everything will go though the slug. So you can remove the /:id endpoint and remove its tests

Reply:
Removed endpoint, adjusted tests. Method on service is used in update endpoint so that still exists.